---
title: "&#39; Try &#39; 종료 내에 나타날 수는 &#39; Try &#39; 문"
ms.date: 07/20/2015
ms.prod: .net
ms.technology: devlang-visual-basic
ms.topic: article
f1_keywords:
- vbc30393
- bc30393
helpviewer_keywords: BC30393
ms.assetid: b8651df3-a32f-478c-a6d8-aa0ef584155f
caps.latest.revision: "7"
author: dotnet-bot
ms.author: dotnetcontent
ms.openlocfilehash: 7eed9e67e08c131a02ff1439c7ac37b1d67049ca
ms.sourcegitcommit: 34ec7753acf76f90a0fa845235ef06663dc9e36e
ms.translationtype: MT
ms.contentlocale: ko-KR
ms.lasthandoff: 12/21/2017
---
# <a name="39exit-try39-can-only-appear-inside-a-39try39-statement"></a>&#39; Try &#39; 종료 내에 나타날 수는 &#39; Try &#39; 문
`Exit Try` 는 `Try` 블록 문에서만 나올 수 있습니다. 중복 `Exit Try` 문이 있거나 또는 `Exit Try` 문이 해당 `Try` 블록의 경계를 벗어난 곳에 나타납니다.  
  
 **오류 ID:** BC30393  
  
## <a name="to-correct-this-error"></a>이 오류를 해결하려면  
  
1.  불필요한 `Exit Try` 문을 찾아서 제거합니다.  
  
2.  이동 된 `Exit Try` 문을 코드 내에서 적절 한 위치로 합니다.  
  
## <a name="see-also"></a>참고 항목  
 [Try...Catch...Finally 문](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)  
 
